Update icon/image documentation to be more clear (#137)
MapsIndoors/mapsindoors.github.io: collections/_cms/index.md

* Edit the Venue name in the available languages
* Choose a default floor

> * Add an image of the Venue. Upload in a 600w x 300h size to make it look its best in our Standard Apps.

Reviewer: Where in the CMS is it possible to add a venue image?

Author: Good catch!

* Edit the Venue name in the available languages
* Choose a default floor

> _A note about Venue images:_ You can get MapsPeople to add an image of the Venue. It should be in a 600w x 300h size to make it look its best in our Standard Apps.

Reviewer 2: ```suggestion
_A note about Venue images:_ You can request MapsPeople to add an image of the Venue. It should be in a 600w x 300h size to make it look its best in our Standard Apps.
```

Anywhere in the CMS you can change icons for one or more markers on the map, you use the Icon Manager to do so. Click the "Change"-button to invoke it.

In the Icon Manager, you can choose between a predefined set of MapsIndoors icons, and a list of your uploaded icons. We support uploads of files in the PNG and SVG formats. Click on an icon in either place, and set it to close the icon manager and preview the icon on the map. Don't worry, it won't be saved until you click "Save" in the details editor.

> Remember to keep your icons at a reasonable size. We'll let you know if it's larger than 128x64px, as that would take up a large amount of space on the map. There is a hard limit on the size of 900x600px or 150kb. The icon files should also be optimized and compressed to be as small as possible.

Reviewer 2: ```suggestion
Remember to keep your icons at a reasonable size. The Icon Manager will prompt you if an icon larger than 128x64px, as that would take up a large amount of space on the map. There is a hard limit on the size of 900x600px or 150kb. The icon files should also be optimized and compressed to be as small as possible.
```

In the Icon Manager, you can choose between a predefined set of MapsIndoors icons, and a list of your uploaded icons. We support uploads of files in the PNG and SVG formats. Click on an icon in either place, and set it to close the icon manager and preview the icon on the map. Don't worry, it won't be saved until you click "Save" in the details editor.

> Remember to keep your icons at a reasonable size. The Icon Manager will prompt you if an icon larger than 128x64px, as that would take up a large amount of space on the map. There is a hard limit on the size of 900x600px or 150kb. The icon files should also be optimized and compressed to be as small as possible.

Author: ```suggestion
Remember to keep your icons at a reasonable size. The Icon Manager will prompt you if an icon is larger than 128x64px, as that would take up a large amount of space on the map. There is a hard limit on the size of 900x600px or 150kb. The icon files should also be optimized and compressed to be as small as possible.
```
